### PR TITLE
EBODAC-464 "QA - Confirmation is needed before un-enrollment & enroll…

### PR DESCRIPTION
**Pull request by tomaszmueller**
…ment Test"

### CODE REVIEW

**Comment by tomaszmueller** [comment]
test this please

**Comment by pmuchowski** [comment]
test this please

**Comment by tomaszmueller** [comment]
test this please

**Comment by pmuchowski** [comment]
test this please

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ConfirmationBeforeUnenrollmentEnrollmentTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import org.motechproject.ebodac.uitest.helper.UITestHttpClientHelper;
+import org.motechproject.ebodac.uitest.page.EBODACPage;
+import org.motechproject.ebodac.uitest.page.EnrollmentPage;
+import org.motechproject.ebodac.uitest.page.HomePage;
+import org.motechproject.ebodac.uitest.page.ParticipantEditPage;
+import org.motechproject.uitest.page.LoginPage;
+
+import static org.junit.Assert.assertTrue;
+
+public class ConfirmationBeforeUnenrollmentEnrollmentTest extends TestBase {
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ConfirmationBeforeUnenrollmentEnrollmentTest.java]: change the name of the test, it should end with "UiTest"

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ConfirmationBeforeUnenrollmentEnrollmentTest.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    private LoginPage loginPage;
+    private HomePage homePage;
+    private EnrollmentPage enrollmentPage;
+    private EBODACPage ebodacPage;
+    private ParticipantEditPage participantEditPage;
+    private UITestHttpClientHelper httpClientHelper;
+    private String url;
+    private String user;
+    private String password;
+    @Before
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ConfirmationBeforeUnenrollmentEnrollmentTest.java]: add a new line here

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ConfirmationBeforeUnenrollmentEnrollmentTest.java [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        if (url.contains("localhost")) {
+            httpClientHelper = new UITestHttpClientHelper(url);
+            httpClientHelper.addParticipant(new TestParticipant(), user, password);
+        }
+        if (homePage.expectedUrlPath() != currentPage().urlPath()) {
+            loginPage.goToPage();
+            loginPage.login(user, password);
+        }
+    }
+    @Test
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ConfirmationBeforeUnenrollmentEnrollmentTest.java]: add empty line before here

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ConfirmationBeforeUnenrollmentEnrollmentTest.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        Thread.sleep(2000);
+        if (enrollmentPage.error()) {
+            enrollmentPage.clickOK();
+            enrollmentPage.nextAction();
+        }
+        Thread.sleep(2000);
+        checkPopUp("Participant was enrolled successfully.", "Participant was unenrolled successfully.");
+        enrollmentPage.clickOK();
+    }
+    @After
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/test/ConfirmationBeforeUnenrollmentEnrollmentTest.java]: same here

**Comment by tomaszmueller** [comment]
test this please

**Pull request merged**